cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,22 @@
+describe('template spec', () => {
```

Comment:
Que tal um nome mais significativo para a suíte de testes?

Ao dar um bom nome para a suíte de testes, fica mais fácil entender do que tal suíte se trata sem a necessidade de ler os detalhes de implementação.

Segue uma sugestão abaixo.

```suggestion
describe('EngageSphere', () => {
```